Mage.Sets/src/mage/cards/t/TheAesirEscapeValhalla.java

```diff
@@ -83,7 +83,7 @@ public boolean apply(Game game, Ability source) {
         controller.choose(outcome, target, source, game);
         Card card = game.getCard(target.getFirstTarget());
         if (card != null) {
-            UUID exileId = CardUtil.getExileZoneId(game, source, 1);
```

Comment:
Is it affected?

![shot_250601_081619](https://github.com/user-attachments/assets/2ad532ab-eaf0-469b-8bc3-478fea51c3ec)

Reply:
`TheAesirEscapeValhallaTest` confirms that the behavior is correct (thanks @Susucre for adding all those tests)

Reply:
I'm talking about method's docs and use cases in it

Reply:
The docs don't need any changing, this only affects the zcc of abilities that trigger as a permanent is entering. The current need for an offset in that case is not mentioned, and with this PR merged that need is removed.